src/server/envelop/index.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import type { FormatErrorHandler } from '@envelop/core'
+import {
+  enableIf,
+  envelop,
+  useExtendContext,
+  useMaskedErrors,
+  useSchema,
+} from '@envelop/core'
+import { useGraphQlJit } from '@envelop/graphql-jit'
+import { useParserCache } from '@envelop/parser-cache'
+import { useResponseCache } from '@envelop/response-cache'
+import { useValidationCache } from '@envelop/validation-cache'
+
+import { getSchema, getContextFactory } from '..'
+
+const { NODE_ENV, CONTEXT: ENV = NODE_ENV } = process.env
+const isProduction = ENV === 'production'
+
+export type FormatError = FormatErrorHandler
+
+export const getEnveloped = async (formatError: FormatError) =>
+  envelop({
+    plugins: [
+      useSchema(await getSchema()),
+      useExtendContext(getContextFactory()),
+      enableIf(isProduction, () => useMaskedErrors({ formatError })),
```

Comment:
Isn't it better to always mask the errors and console.log it instead so people don't use error respose on the frontend as logic?

Reply:
Makes sense. The error's already being logged at dev 👍